Changelog.md

```diff
@@ -8,11 +8,13 @@ Language Features:
 
 Compiler Features:
  * Command Line Interface: Do not perform IR optimization when only unoptimized IR is requested.
+ * Commandline Interface: Output transient storage layout with option ``--transient-storage-layout``.
  * Error Reporting: Unimplemented features are now properly reported as errors instead of being handled as if they were bugs.
  * EVM: Support for the EVM version "Prague".
  * SMTChecker: Add CHC engine check for underflow and overflow in unary minus operation.
  * SMTChecker: Replace CVC4 as a possible BMC backend with cvc5.
  * Standard JSON Interface: Do not perform IR optimization when only unoptimized IR is requested.
+ * Standard JSON Interface: Output the transient storage layout of a contract when artifact ``transientStorageLayout`` is requested.
```

Comment:
How about this? More compact and we generally call these "outputs" rather than "artifacts" (at least when referring to the the option/JSON key itself).

```suggestion
 * Commandline Interface: Add ``--transient-storage-layout`` output.
 * Error Reporting: Unimplemented features are now properly reported as errors instead of being handled as if they were bugs.
 * EVM: Support for the EVM version "Prague".
 * SMTChecker: Add CHC engine check for underflow and overflow in unary minus operation.
 * SMTChecker: Replace CVC4 as a possible BMC backend with cvc5.
 * Standard JSON Interface: Do not perform IR optimization when only unoptimized IR is requested.
 * Standard JSON Interface: Add ``--transientStorageLayout`` output.
```